apps/cowswap-frontend/src/modules/trade/containers/TradeWidget/TradeWidgetForm.tsx

```diff
@@ -98,6 +125,13 @@ export function TradeWidgetForm(props: TradeWidgetProps) {
           {isInjectedWidgetMode && !injectedWidgetParams.hideConnectButton && (
             <AccountElement isWidgetMode={isInjectedWidgetMode} pendingActivities={pendingActivity} />
           )}
+
+          {isUpToLarge && (
+            <ButtonOutlined margin={'0 16px 0 auto'} onClick={handleClick}>
+              My orders <SVG src={ICON_ORDERS} />
+            </ButtonOutlined>
+          )}
```

Comment:
As Elena noticed, we should not show this on recreate/edit order.
I can help you with this.

Reply:
@alfetopito This has been addressed now